instructors.md

```diff
@@ -156,9 +199,143 @@ A typical schedule for a two-day course is:
     *   15:45 Open discussion
     *   16:15 Wrap up and minute cards for afternoon session
 
+## Why We're Not a MOOC
+
+> If you use robots to teach, you teach people to be robots.
+
+This difference between what novices are doing when they learn,
+and what competent practitioners are doing,
+is one of the reasons we have stopped trying to teach via recorded video
+with auto-graded drill exercises.
+Any recorded content is as ineffective for most learners as broadcast television,
+or as a professor standing in front of 400 people in a lecture hall,
+because neither can intervene to clear up specific learners' misconceptions.
+Some people happen to already have the right conceptual categories for a subject,
+or happen to form them correctly early on;
+these are the ones who stick with most massive online courses,
+but many discussions of the effectiveness of such courses
+ignore this survivor bias.
+
+## Why So Much Documentation Is Useless
+
+The transition from novice to competent practitioner is no more
+and no less than
+the construction of correct (enough) categories,
+i.e.,
+the construction of a new mental model of this new intellectual domain.
+The goal of education for novices is, therefore,
+to help them form the right categories.
+Until they've done that,
+trying to impart "mere information" just confuses them.
+
+This is one of the reasons software documentation is so often frustrating.
+Reference material is opaque to someone who doesn't know what they're looking for,
+such as a novice who doesn't yet have a mental map of the domain.
+On the other hand,
+tutorials meant to help people build such a map
+are too slow and too diffuse for people who already have one.
+It is possible to craft something that serves both communities,
+but it's often simpler to address their needs separately.
+
+As an example of this idea's practical implications,
+our introduction to the Unix shell only introduces 12 commands,
+and we usually spend two and a half hours working through it.
+Twelve and a half minutes per command may seem glacially slow,
+but that's because our real goal isn't to teach specific commands:
+it's to help learners construct the mental model and notional machine
+that these commands fit into.
```

Comment:
As an alternative heading in this section, perhaps the main point is “our lessons are slower than expert-user docs because we take time to scaffold the required mental models”, in which case a good heading might be “Why we go slow” or “Why we prioritize mental models over deep dives into tooling”.